src/main/java/com/github/dockerjava/api/DockerClient.java

```diff
@@ -121,8 +123,37 @@
 
     public LogContainerCmd logContainerCmd(@Nonnull String containerId);
 
+    /**
+     * Copy resource from container to local machine.
+     *
+     * @param containerId id of the container
+     * @param resource    path to container's resource
+     * @return created command
+     * @since docker API version 1.20 and above.
+     */
+    public CopyArchiveFromContainerCmd copyArchiveFromContainerCmd(@Nonnull String containerId, @Nonnull String resource);
+
+    /**
+     * Copy resource from container to local machine.
+     *
+     * @param containerId id of the container
+     * @param resource    path to container's resource
+     * @return created command
+     * @see #copyArchiveFromContainerCmd(String, String)
+     * @deprecated since docker API version 1.20, replaced by {@link #copyArchiveFromContainerCmd(String, String)}
```

Comment:
(Not critical, but for simplification there are API versions for linking javadocs. Example https://github.com/docker-java/docker-java/blob/58d94079d2cd7ae7d072e5aad599777707e5b10a/src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java#L234 )